### ISSUE TITLE
Image reference in markdown to latex

### ISSUE THREAD

**Issue by ConsoleTVs** [jgm/pandoc#2785]
Hello, i am facing a problem, reference an image!

I found something like this:

```
[image]: https://i.ytimg.com/vi/gxmHOwWh-i4/maxresdefault.jpg "Image Title"

![Alt text][image]

A reference to the [image](#image).
```

But the link from the reference just makes it go to Page 0. Any solutions?

**Comment by jgm** [comment]
I'm not sure what you're trying to do.  Pandoc produces this HTML, which seems correct:
``` html
<div class="figure">
<img src="https://i.ytimg.com/vi/gxmHOwWh-i4/maxresdefault.jpg" title="Image Title" alt="Alt text" />
<p class="caption">Alt text</p>
</div>
<p>A reference to the <a href="#image">image</a>.</p>
```
What were you expecting?

**Comment by ConsoleTVs** [comment]
@jgm I'm talking about pdf conversion with latex, sorry for not telling it! I want to add an image and later or before create a reference to it, making a link clickable and go to the image, its the same as the section references

**Comment by jgm** [comment]
If you link to `#image`, you need to define an anchor for it to link to.

With recent pandoc, you can do this:

    ![Alt text][image]{#image}

and then

    A reference to the [image](#image)

**Comment by ConsoleTVs** [comment]
@jgm Hello, first in markdown i guess you misstyped `![Alt text][image]{#image}``willing to type `![Alt text](image-url){#image}`

But in any case, i rendered some test files for you:

Here there's the source .md file: <https://www.dropbox.com/s/2b5epmeerq4o408/test.md?dl=1>

And here the result: <https://www.dropbox.com/s/dl7vst9959c5ac5/test.pdf?dl=1>

If you press the IMAGE text on the second page you'll get redirected to the first page without a reason, while if you press the THIS text, you'll get redirected to the heading correctly.

**Comment by jgm** [comment]
Ah yes.  We need PR #2637 to make this work.

For now, though, you could wrap the image in a `<div id="image">` and it should work.

**Comment by jgm** [comment]
OK, this is now fixed by the merge of #2637.
